datawig/imputer.py

```diff
@@ -85,6 +87,10 @@ def __init__(self,
         self.precision_recall_curves = {}
         self.calibration_info = {}
 
+        self.is_explainable = np.any([isinstance(encoder, CategoricalEncoder) or
+                                      isinstance(encoder, TfIdfEncoder)
+                                      for encoder in self.data_encoders])
+
```

Comment:
we should also check the label encoder to avoid trying to explain models with numeric targets for now.